src/python/pants/backend/scala/target_types.py

```diff
@@ -49,6 +50,16 @@ class ScalaDependenciesField(Dependencies):
     pass
 
 
+class ScalaConsumedPluginNamesField(StringSequenceField):
+    """The names of Scala plugins that this source file requires.
+
+    If not specified, this will default to the plugins specified in `--scalac-plugins` for this
+    target's resolve.
+    """
+
+    alias = "experimental_plugins"
```

Comment:
Should this be called `scalac_plugins` instead? That would identify what kind of plugin this is.

Also is the "experimental" prefix necessary? Not sure what naming convention we are using when the whole backend is experimental. I haven't been prefixing things with "experimental."

Reply:
> Also is the "experimental" prefix necessary?

Yeah we don't want it. Unnecessary churn for users when they'll have to update their BUILD files.